napari/_qt/_qapp_model/qactions/__init__.py

```diff
@@ -0,0 +1,31 @@
+from functools import lru_cache
+from typing import Optional
+
+# Submodules should be able to import from most modules, so to
+# avoid circular imports, don't import submodules at the top level here,
+# import them inside the init_qactions function.
+
+
+@lru_cache  # only call once
+def init_qactions():
```

Comment:
Sorry if this is captured elsewhere, but I am curious about the rationale dividing the qactions from the actions. It seems that actions that involve qt are brought here, whereas the ones that are not qt related are stored under actions. Is there any advantage separating the two? (trying to port the help menu and realized it also covers both category)

Reply:
(also by any chance we can unify the naming convention? right now it is _view_actions.py vs _view.py. If they serve a similar function I would either call them both _view.py or _view_actions.py and _view_qactions.py)

Reply:
Don't want to have to import Qt related stuff outside of our Qt folder.  So, unlike the other actions (like layer actions), these are all things that perform gui related tasks or need Qt to perform them. 

I don't think, at this point, we need to have any particular rules or expectations on where actions get defined.  Let's take the refactoring one step at a time